assemble/Momentum_DG.F90

```diff
@@ -589,12 +592,52 @@ subroutine construct_momentum_dg(u, p, rho, x, &
        FLAbort("Unknown viscosity scheme - Options tree corrupted?")
     end if
 
-    partial_stress = have_option(trim(u%option_path)//&
+    partial_stress = .false.
+    have_les = .false.
+    if (have_option(trim(u%option_path)//&
          &"/prognostic/spatial_discretisation"//&
          &"/discontinuous_galerkin/viscosity_scheme"//&
-         &"/partial_stress_form")
+         &"/partial_stress_form")) then
+
+      partial_stress = .true.
+      
+      ! if we have stress form then we may be doing LES modelling
```

Comment:
This silently ignores the new discontinuous_galerkin/les_model option if you haven't also selected partial_stress_form. At the very least there should be an options check to ensure that doesn't happen. Even with that check there's no reason to write the logic like that here.